docs/_instructions/namesection.md

```diff
@@ -5,78 +5,73 @@ title: -namesection RESOURCE-SPEC ( ',' RESOURCE-SPEC ) *
 summary:  Create a name section (second part of manifest) with optional property expansion and addition of custom attributes. 
 ---
 
-	/**
-	 * Parse the namesection as instructions and then match them against the
-	 * current set of resources For example:
-	 * 
-	 * <pre>
-	 * 	-namesection: *;baz=true, abc/def/bar/X.class=3
-	 * </pre>
-	 * 
-	 * The raw value of {@link Constants#NAMESECTION} is used but the values of
-	 * the attributes are replaced where @ is set to the resource name. This
-	 * allows macro to operate on the resource
-	 */
+Create a name section (second part of manifest) with optional property expansion and addition of custom attributes.
 
-	private void doNamesection(Jar dot, Manifest manifest) {
+### Matching
+The key of the `-namesection` instruction is an _ant style_ glob. And there are two target groups for matching:
 
-		Parameters namesection = parseHeader(getProperties().getProperty(NAMESECTION));
-		Instructions instructions = new Instructions(namesection);
-		Set<String> resources = new HashSet<String>(dot.getResources().keySet());
+* **resources** - the pattern ends with `*` or is an exact match for a resource path
```

Comment:
This would be a pattern not ending with `/` (vs. a pattern ending in `*`).

Reply:
fixed